calyx-py/calyx/queue_call.py

```diff
@@ -66,20 +68,19 @@ def insert_runner(prog, queue, name, num_cmds, use_ranks, stats_component=None):
     cmd = runner.reg(2)  # The command we're currently processing
     value = runner.reg(32)  # The value we're currently processing
     rank = runner.reg(32)  # The rank we're currently processing
-    
-    load_data = []
-    if use_ranks:
-        load_data = [
-            runner.mem_load_d1(commands, i.out, cmd, "write_cmd"),  # `cmd := commands[i]`
-            runner.mem_load_d1(values, i.out, value, "write_value"),  # `value := values[i]`
-            runner.mem_load_d1(ranks, i.out, rank, "write_rank"),  # `rank := ranks[i]`
+
+    load_data = (
+        [  # `cmd := commands[i]`, `value := values[i]`, `rank := ranks[i]`
+            runner.mem_load_d1(commands, i.out, cmd, "write_cmd"),
+            runner.mem_load_d1(values, i.out, value, "write_value"),
+            runner.mem_load_d1(ranks, i.out, rank, "write_rank"),
         ]
-    else:
-        load_data = [
-            runner.mem_load_d1(commands, i.out, cmd, "write_cmd"),  # `cmd := commands[i]`
-            runner.mem_load_d1(values, i.out, value, "write_value"),  # `value := values[i]`
+        if use_ranks
+        else [  # `cmd := commands[i]`, `value := values[i]`
+            runner.mem_load_d1(commands, i.out, cmd, "write_cmd"),
+            runner.mem_load_d1(values, i.out, value, "write_value"),
         ]
```

Comment:
Sorry the diff is large; we have sightly different linter or formatter settings I think. Anyway this is the change I meant to make.